src/IOCapture.jl

```diff
@@ -62,10 +66,17 @@ function iocapture(f; throwerrors::Union{Bool,Symbol}=true)
     # Redirect both the `stdout` and `stderr` streams to a single `Pipe` object.
     pipe = Pipe()
     Base.link_pipe!(pipe; reader_supports_async = true, writer_supports_async = true)
-    redirect_stdout(pipe.in)
-    redirect_stderr(pipe.in)
+    if VERSION >= v"1.6.0-DEV.481" # https://github.com/JuliaLang/julia/pull/36688
```

Comment:
Worth sticking a `@static` in here to avoid the runtime branch?

Reply:
I'm skeptical that there is a runtime branch, but I'll stick the `@static` to be clear.

Reply:
Yeah, looks like it's smart enough these days:

```
julia> function f(x, y)
           if VERSION >= v"1.4.0"
               x + y
           else
               x - y
           end
       end

julia> @code_typed f(1, 2)
CodeInfo(
1 ─ %1 = Base.add_int(x, y)::Int64
└──      return %1
) => Int64
```

No point adding `@static` then.